stanford/roles/cluster/tasks/main.yml

```diff
@@ -91,6 +91,7 @@
     subnets:
       - "{{ CLUSTER_SUBNET_ID }}"
     purge_subnets: no
+    idle_timeout: "{{ item.idle_timeout | default(60) }}"
```

Comment:
@dcadams I updated this so that it still defaults to 60 seconds (AWS' default),
but can be overridden on a case-by-case basis, here only `edxapp`.